### ISSUE TITLE
Unknown attribute batterySupport with value enabled

### ISSUE THREAD

**Issue by ColinRobbins** [tillsteinbach/WeConnect-python#67]
**Describe the bug**
When running the example `allVehicles.py` I get an error message.

**To Reproduce**
Run `allVehicles.py`

**Expected behavior**

**Screenshots**
The example should run without error.

**Logs**
```
/vehicles/XXXXX/domains/batterySupport/batterySupportStatus: Unknown attribute batterySupport with value enabled

```

**Additional context**
The error message causes issues with downstream use of the library, such as https://github.com/mitch-dc/volkswagen_we_connect_id

**Comment by tillsteinbach** [comment]
Hi @ColinRobbins: Sorry, I cannot reproduce this with my car. Do you have an ID with SW Version 2.4 or 3.0? Are these all messages you are getting?

**Comment by ColinRobbins** [comment]
Yes, they are the only messages I am seeing - everything else is OK.
How do I tell which S/W version?  I am guessing the latest as I’ve only taken delivery in the last few weeks.
Happy to try and produce debugging info, if guided as to what you need.

**Comment by tillsteinbach** [comment]
I added the missing attribute. Could you give the example a new try using the main branch?

**Comment by ColinRobbins** [comment]
Yes, this works in my test environment.
Thank you.

I doing so, I found a few dependancies missing, I'll make a PR shortly.

**Comment by tillsteinbach** [comment]
Perfect, I will make that part of the next release then.
Regarding the dependencies I answered in the PR.

**Comment by ColinRobbins** [comment]
Hi. 
Did this make it into 0.45.1?
I am still seeing the error in 0.45.1, but not when I use your main branch.

**Comment by tillsteinbach** [comment]
No, will be part of 0.45.2. My car is currently in the workshop for the 2.4 update. When I have it back I can prepare an update

**Comment by tillsteinbach** [comment]
This is released in 0.46.0
